pkg/controller/tuned/tuned_controller.go

```diff
@@ -66,6 +66,12 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	createCustomResource(mgr)
```

Comment:
- Why did you pull this into a function?
- Do we want to have the operator create its own CR?